### PR TITLE
Enforce import type for the type imports

### PR DESCRIPTION
**Pull request by BatDroid**
### Summary
related to [this issue](https://github.com/callstack/react-native-paper/issues/1996).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,8 +67,8 @@
     "@typescript-eslint/parser": "^2.12.0",
     "all-contributors-cli": "^6.1.1",
     "babel-cli": "^6.26.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-jest": "^24.3.1",
+    "babel-core": "^6.26.3",
```
Comment (Trancever) [inline package.json:70]: Why is it downgraded?

Comment (BatDroid) [reply package.json:70]: maybe it has happened after conflict resolve. I'll fix it now.

Comment (Trancever) [reply package.json:70]: Awesome, thanks a lot 🙏

Comment (BatDroid) [reply package.json]: Pushed. And thank you for letting me doing it. Hope I'll be able to contribute more  :)

**Pull request merged**